src/pages/admin/reports/components/program-expenses-section/components/program-expenses-toolbar/ProgramExpensesToolbar.module.scss

```
@@ -0,0 +1,142 @@
+    }
+
+    svg {
+        width: 26px;
+        height: 26px;
+        flex-shrink: 0;
+    }
+}
+
+.program-option {
+    @include type.body-2-regular;
+    min-height: 56px;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)